woocommerce/woocommerce
# Cart/Checkout: Adjust styling of product attributes so that they are visually distinct.

### Changes proposed in this Pull Request:

Closes #42022 

* Adds a small amount of margin to each product description item.
* Always apply bold styling to the `name` element.



### How to test the changes in this Pull Request:

This is a visual change and also impacts cart, so check that it hasn't introduced any visual regressions or issues in either cart or checkout.

1. Add a product with some variations to your store, ideally more than one so you can see the list displayed.
2. As a shopper add some of the variable products to your cart
3. Examine the styling of the list of product attributes in cart and checkout

### Before:

#### Mobile:

<img width="216" alt="Screenshot 2024-11-06 at 7 14 42 PM" src="https://github.com/user-attachments/assets/514c505f-bcfc-4eed-b00e-41c4c1aecf03">

#### Desktop:

<img width="432" alt="Screenshot 2024-11-06 at 7 15 47 PM" src="https://github.com/user-attachments/assets/0bb9a35f-b1e1-446c-8369-8cfc820a1ed1">


### After:

#### Mobile:
<img width="484" alt="Screenshot 2024-11-06 at 4 47 54 PM" src="https://github.com/user-attachments/assets/76460d04-679d-48ff-83d8-9c6c41876f43">

#### Desktop:
<img width="494" alt="Screenshot 2024-11-06 at 4 46 43 PM" src="https://github.com/user-attachments/assets/86a08214-e141-453f-a689-ce899a4f6f3d">





### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Cart/Checkout: Adjust styling of product attributes so that they are visually distinct.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
